### PR TITLE
New Formit RU translations - for validators, country/state selectors + couple of fixes

### PR DESCRIPTION
**Pull request by ant-tar**
## Description

Formit RU translation for validators, country/state selectors + couple of fixes

## Affected versions

both 2.x. 3.x.

### CODE REVIEW

File: redirects.json [hunk 1]
```diff
@@ -2297,5 +2297,6 @@
     "/display/xPDO20/xPDO.getCollectionGraph": "en/extending-modx/xpdo/retrieving-objects/graphs",
     "/display/ADDON/Login.UpdateProfile": "en/extras/login/login.updateprofile",
     "/display/ADDON/EvoForcedPasswdChangeGallery": "en/extras/forcedpasswdchange",
-    "/display/ADDON/FormIt.FormItRetriever": "en/extras/formit/formit.formitretriever"
+    "/display/ADDON/FormIt.FormItRetriever": "en/extras/formit/formit.formitretriever",
+	"/display/revolution20/display/revolution20/Setting+up+a+Development+Environment": "en/extending-modx/development-environment"
```
Comment (Ibochkarev) [inline redirects.json]:
```suggestion
    "/display/revolution20/display/revolution20/Setting+up+a+Development+Environment": "en/extending-modx/development-environment"
```

**Review by Ibochkarev — CHANGES_REQUESTED** [review]

**Comment by ant-tar** [comment]
Thanks @Ibochkarev ! 
The only issue is the same as before: https://docs.modx.com/current/ru/extras/formit page exists, but left menu hasn't been updated yet. @Mark-H could you please recheck that again. Index file exists now.

**Comment by Mark-H** [comment]
Can you open a separate issue for that? I'm really busy and will forget otherwise.

**Comment by Mark-H** [comment]
I don't immediately see why that wouldn't work, index is there like you say.

**Comment by ant-tar** [comment]
> 
> 
> Can you open a separate issue for that? I'm really busy and will forget otherwise.

Thanks @Mark-H I'll create it!

**Pull request merged**